Getting Input From User

In [4]:
x = input("Enter First Number")
y = input("Enter Second Number")
x+y

Enter First Number25
Enter Second Number25


'2525'

Here the above code gives 2525 instead of giving 25 + 25 = 50 that is because of we are getting input as string we need to convert string to integer to perform addition operations

In [3]:
a = int(input("Enter First Number"))
b = int(input("Enter Second Number"))
a+b

Enter First Number25
Enter Second Number25


50

In [5]:
a1 = int(x)
a2 = int(y)
a1+a2

50

In [8]:
x = input("Give any word ")
print(x[0])

Give any word Vikram
V


In [11]:
x = input("Enter Float number 1 : ")
y = input("Enter Float Number 2 : ")
a1 = float(x)
a2 = float(y)
a1+a2

Enter Float number 1 : 5.5
Enter Float Number 2 : 5.5


11.0

In [12]:
result = eval(input("Enter number"))
print(result)

Enter number4+6-2+7
15
